sections/introduction.tex

```diff
@@ -1,32 +1,70 @@
-approaches to teaching the skills and ideas needed to do science. This is
-particularly true in teaching computational and data analytic techniques,
+Over the last several years science has seen a large growth in the use of
+computationally intensive and data-centric methods. Researchers are
+increasingly using programming languages such as R or Python, and utilizing
+complex algorithms in applied statistics and machine learning in order to
+perform their research.
+
+With an increased focus on computational methods
+comes new challenges in teaching these techniques and sharing
+knowledge with fellow scientists. There has been a growing interest in learning
+outside of the traditional classroom setting. This is particularly true
+for teaching computational and data analytic techniques,
 as instructors must teach both conceptual and methodological information
-at the same time. One common way to teach this type of skills is a short,
+simultaneously. One common way to teach this type of skills is a short,
 time-bounded learning event, such as a bootcamp or short-course
 \cite{wilson2016software}. These attempt to compress several
 topics into an intensive learning session that is usually held over one or
 several days.
 
-Workshops often follow the same formula, roughly described here: First,
-instructors develop materials on their own computers, sharing them with
+As these new models for learning are being adopted, it offers
+opportunities in developing new technology and models for pedagogy that is
+focused around "hands-on" learning. Teaching a class in which students
+must simultaneously learn conceptual information from the instructor, but also
+perform work on their own laptops, poses a number of challenges for the
+current standard approach to the bootcamp model of learning. Here we describe
+recent work in utilizing cloud-based infrastructure to enhance the learning
+experience of students, and streamline the ability of instructors to
+prepare materials that focus on data analytic techniques. Our approach
+utilizes recent advances in cloud- and cluster- based technology, such
+as Kubernetes, Container technology, and the Jupyter framework for computing.
+
+This article is a case-study covering our recent experience implementing this approach, using
+advanced cyberinfrastructure to teach a day-long bootcamp in machine learning at
+the University of California in San Francisco. We describe the
+technical tool-chain and processes that were used in designing this course. This
+unique design includes hosting all course materials online and providing
+an interactive, online environment where students can run code via the cloud
+without requiring them to download anything onto their machine.
+
+Importantly, this stack only utilizes tools that are
+open-source and freely available to
+the community, uses computing services that are available at a low-cost
+to instructors and institutions, and requires only a single course organizer
```

Comment:
I went with "Importantly, the
use of these technologies does not require that the entire
team of instructors become fully skilled at using them. It is
usually enough for a single team member to set up and manage these resources
for everyone to use."

ok?